.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "src/common/thirdparty/redis"]
+	path = src/common/thirdparty/redis-windows
+	url = https://github.com/MSOpenTech/redis.git
+	ignore = all
+[submodule "src/common/thirdparty/python"]
+	path = src/common/thirdparty/python
+	url = https://github.com/austinsc/python.git
+	ignore = all
```

Comment:
I assume you mean `git clone` and not `wget`...
It's of course possible... but it just makes life hard. Why?